README.md

```diff
@@ -1,2 +1,36 @@
-# reactive-streams-scalajs
-Reactive Streams bindings for ScalaJS
+# Reactive Streams for Scala.js
+
+The purpose of Reactive Streams is to provide a standard for asynchronous stream processing with non-blocking backpressure.
+
+This repository manages the [Scala.js](http://www.scala-js.org/)
+builds of the API. For details and the TCK see
+[the JVM](https://github.com/reactive-streams/reactive-streams-jvm).
```

Comment:
TCK should likely be ported too though, since we'll want to check the implementations against it?
The concurrency things aside probably since `js`